pkg/scale/encode_test.go

```diff
@@ -909,9 +909,28 @@ var (
 		},
 	}
 
+	mapTests = tests{
+		{
+			name: "testMap1",
+			in:   map[int8][]byte{2: []byte("some string")},
+			want: []byte{4, 2, 44, 115, 111, 109, 101, 32, 115, 116, 114, 105, 110, 103},
```

Comment:
For both encoding and decoding, were any of the expected values derived from substrate? Or were the all derived from our own scale implementation?

Reply:
yes all derived from substrate.